_docs/_latest/configure/instrumentation/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Instrumentation 
+order: 200
+---
+
+# {{ page.title }}
+
+OpenCensus tracing has been added to Forseti as part of instrumentation on 
+Forseti. 
+
+---
+
+## Configuring Instrumentation
+
+**Tracing**
+* Tracing is enabled by default and the traces are automatically 
+exported to `Stackdriver Trace`.
+* If you wish to disable tracing, you may do so by modifying the 
+`initialize_forseti_services.sh`script found under `install/gcp/scripts` to 
+remove `FORSETI_COMMAND+=" --enable tracing"` and redeploy Forseti. 
+* Tracing is done in a background thread.
```

Comment:
Do we have a page on how to use instrumentation?  Because lines 21-23 seems more appropriate for Use rather than Configuration.